tests/benchmarks/cmsis_dsp/basicmath/src/f32.c

```diff
@@ -348,18 +348,12 @@ void test_benchmark_vec_dot_prod_f32(void)
 	TC_PRINT(BENCHMARK_TYPE " = %u\n", timespan);
 }
 
-void benchmark_basicmath_f32(void)
-{
-	ztest_test_suite(basicmath_f32_benchmark,
```

Comment:
we have some very primitive scanning in twister to identify testcases, see https://github.com/zephyrproject-rtos/zephyr/blob/718c77a4cdc4aa0e3b31e51feb15f1d7638b211f/scripts/pylib/twister/twisterlib.py#L1653 for example. This needs to be adapted to also parse ztest_register_test_suite

Reply:
Thanks for catching this. I had to get a bit creative since the call to register the test suite and the call to run it may not be in the same file and the call to run the suites doesn't specify which tests to run. I effectively looked across all the files to see if tests were registered and if so, required at least one call to run them.

Reply:
I'll add unit tests to twister shortly, just wanted to upload the current setup as early as I could to make sure everyone is happy with the approach.

Reply:
OK, tests were updated and everything seems to be passing on my end.